examples/cookbook/cookbook/urls.py

```diff
@@ -1,9 +1,10 @@
-from django.conf.urls import url
+# from django.conf.urls import url # Deprecated in Django 3.0, see https://stackoverflow.com/a/70319607
```

Comment:
Does this relate to the actual content of the PR? If any maintenance task should be done, please send it in a separate PR.

Reply:
No it does not relate to the PR thank you for pointing this out. I'll re-write the commit history to remove this change and ping you again for review if that's okay

Reply:
@that-one-arab My honest recommendation at this point will be to drop a half dead package and look for a modern alternative. It's nice to see people contributing into therepo, but I choose the same road around 5 years ago and so far only progress the project achieved was to get even more outdated and problematic.

Reply:
I agree but its a time consuming process to migrate existing logic at the moment, I did this feature out of necessity for my own personal project and I hope it helps others as well. Maybe we can deprecate this library and suggest alternatives for it in the README or somewhere similar, its up to you guys I guess...